File: src/main/java/facts/stmt/Assign/MemAssign.java
package facts.stmt.Assign;

import astnodes.exp.Expr;
import astnodes.exp.MemLoad;

/**
 * Store fact
 */
public class MemAssign extends Assign {
    public MemAssign(String pc, MemLoad memExp, Expr rhsExp) {
        super(pc, memExp, rhsExp);
    }
}

File: src/main/java/facts/stmt/Assign/MemAssign.scala
package facts.stmt.Assign

import astnodes.exp.{BinOp, Var, Expr, MemLoad}

/** Store fact
  */
case class MemAssign(override val pc: String, val memExp: MemLoad, val rhsExp: Expr) extends Assign (pc, memExp, rhsExp)  {
}
